src/community/wps-sextante/src/main/java/org/geoserver/wps/sextante/GTDBFIterator.java

```diff
@@ -0,0 +1,53 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wps.sextante;
+
+import java.io.IOException;
+
+import org.geotools.data.shapefile.dbf.DbaseFileReader;
+
+import es.unex.sextante.core.Sextante;
+import es.unex.sextante.dataObjects.IRecord;
+import es.unex.sextante.dataObjects.IRecordsetIterator;
+import es.unex.sextante.dataObjects.RecordImpl;
+import es.unex.sextante.exceptions.IteratorException;
+
+/**
+ * 
+ * @author Cesar Martinez Izquierdo
```

Comment:
Jody, there is a problem here, a module graduation has requirements, code review and IP check are not part of it, the module is supposed to be sane from IP point of view when it's committed to community already.